### ISSUE TITLE
AIL DIFF

### ISSUE THREAD

**Issue by prondel** [FrSky-OS/discussion#20]
From my point of view, AIL DIFFERENTIAL should be a unique value per channel, positive or negative

![image](https://cloud.githubusercontent.com/assets/16406160/11966128/faa7243c-a8f9-11e5-8588-ee6b9be3ead8.png)

**Comment by AloftHobbies** [comment]
Yes, I much prefer the simple setup of a Spektrum when it comes to differential. Check out this short video of their setup:
https://www.youtube.com/watch?v=uXupKQ8-MQs

It would also be nice to have an adjustable input for fine tuning the differential while flying. 

Aileron to flap mix should then use this differential value on the flaps also.
